manifests/local/deployment.yaml

```diff
@@ -3,14 +3,14 @@ nameGenerator:
   suffix:
     valueFrom:
       envVariable: GLOBAL_CUSTOM_DEPLOYMENT_NAME_IDENTIFIER
-toolchainRegion: us-east-1
+toolchainRegion: us-west-2
 groups:
   - name: networking
     path: manifests/local/networking-modules.yaml
   - name: storage
     path: manifests/local/storage-modules.yaml
-  # - name: replication
-  #   path: manifests/local/replicator-modules.yaml
+  - name: replication
```

Comment:
revert?

Reply:
Not needed.

Reply:
I don't understand...if the replication module is not needed, why did you add it in this PR?